tests/libres_tests/job_runner/test_protobuf_reporter.py

```diff
@@ -0,0 +1,235 @@
+import os
+
+import pytest
+from ert.experiment_server._schema_pb2 import (
+    JOB_FAILURE,
+    JOB_RUNNING,
+    JOB_START,
+    JOB_SUCCESS,
+)
+from ert.experiment_server._schema_pb2 import Job as PJob
+from libres_utils import _mock_ws_thread
+
+from job_runner.job import Job
+from job_runner.reporting import Protobuf
+from job_runner.reporting.message import Exited, Finish, Init, Running, Start
+from job_runner.reporting.statemachine import TransitionError
+
+
+def test_report_with_successful_start_message_argument(unused_tcp_port):
```

Comment:
What it is that we test here? Might consider some comments or slight refactoring at least.